README.md

```diff
@@ -107,6 +107,7 @@ To see all rules supported by the makefile, run `make help`
 ### Run controller locally
 ```bash
 make install
+oc project devworkspace-controller
```

Comment:
It would be better to make like the following:
```
export NAMESPACE="devworkspace-controller"
make install
oc patch deployment/devworkspace-controller-manager --patch "{\"spec\":{\"replicas\":0}}" -n $NAMESPACE
make run
```
otherwise it may happen that DWO is deployed in one namespace, but run try to patch in another.